sonar-java-plugin/src/main/resources/static/documentation.md

```diff
@@ -63,6 +63,15 @@ Example: `sonar.java.source=1.6`
 
 If the property is provided, the analysis will take the source version into account, and execute related rules accordingly. At run time, each of these rules will be executed – or not – depending of the Java version used by sources within the project. For instance, on a correctly configured project built with Java 6, rules targeting Java 7 and Java 8 will never raise issues, even though they are enabled in the associated rule profile.
 
+## Batch mode settings
+
+By default, files are parsed in batches. The size of the batch is dynamically computed based on the maximum memory available.
+It is possible to manually set this value by using the property `sonar.java.experimental.batchModeSizeInKB`.
```

Comment:
Now that Batch Mode is the default, I don't think we should keep the word "experimental" in the name of the property.